ui/src/proxy/settings.ts

```diff
@@ -40,9 +41,16 @@ export enum CodeFont {
   System = "system",
 }
 
+export enum FontColor {
+  Blue = "blue",
+  Pink = "pink",
+  Orange = "orange",
+}
+
```

Comment:
This is still useful right? Or should this be somewhere else?

Reply:
I don't think you'll need it.

Reply:
If I don't need that, then I won't have these [here](https://github.com/radicle-dev/radicle-upstream/pull/2380/files?file-filters%5B%5D=.svelte&file-filters%5B%5D=.ts#diff-846decf2ac6490b0e2f40492c83589421eb3567d40b6366fc1cf2d55dcef0cf6R66) either, right?

Reply:
Yes, I think you'll have to reimplement the part which links the SegmentedControl to the store.